eng/signcheck.exclusions.txt

```diff
@@ -0,0 +1,6 @@
+content/sdk/*/AppHostTemplate/apphost.exe;AspNetCoreRuntime.*.nupkg; Exclude the apphost because this is expected to be code-signed by customers after the SDK modifies it.
+content/*.js;Microsoft.DotNet.Web.Spa.ProjectTemplates.*.nupkg; Exclude JavaScript files from codesigning in project templates
+content/*.js;Microsoft.DotNet.Web.ProjectTemplates.*.nupkg; Exclude JavaScript files from codesigning in project templates
+Templates/*.js;Microsoft.VisualStudio.Web.CodeGenerators.Mvc.*.nupkg; Exclude JavaScript files from codesigning in code generators
+*.js;signalr-*-javadoc.jar; Exclude JavaScript files in the generated javadocs
+*.dll|*.exe;IIS.StressTestWebSite.zip; This is a test asset which is only produced for manual upload into IIS stress testing.
```

Comment:
How is this line interpreted? It does _not_ prevent ALL dlls and exes from being signed, right?